src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -60,9 +74,9 @@ public void handlesSocket() throws IOException {
         Mockito.when(socket.getInputStream()).thenReturn(
             new ByteArrayInputStream(
                 Joiner.on("\r\n").join(
-                    "GET / HTTP/1.1",
+                    "POST / HTTP/1.1",
```

Comment:
@dmzaytsev 
In most practical applications, GET method shouldn't be attached with a body.
see latest discussion in #306 for more information.